content/en/docs/reference/glossary/pod-disruption-budget.md

```diff
@@ -0,0 +1,15 @@
+id: pod-disruption-budget
+name: Pod Disruption Budget
+full-link: /docs/concepts/workloads/pods/disruptions/
+aka:
+ - PDB
+related:
+ - pod
+ - container
+tags:
+ - operation
+short-description: >
+ An object that limits the number of Pods of a replicated application, that are down simultaneously from voluntary disruptions.
```

Comment:
```suggestion
 An object that limits the number of {{< glossary_tooltip text="Pods" term_id="pod" >}} of a replicated application, that are down simultaneously from voluntary disruptions.
```

I think this kind of cross-reference is helpful; however it's also broken right now (see #12608).

Reply:
@sftim I agree. Thank you!